src/encoded/tests/test_schema_library.py

```diff
@@ -28,6 +28,34 @@ def library_starting_quantity_no_units(library):
     })
     return item
 
+@pytest.fixture
+def library_fragmentation_method_string(library):
+    item = library.copy()
+    item.update(
+        {
+            'fragmentation_method': 'chemical (DpnII restriction)',
+        }
+    )
+    return item
+
+@pytest.fixture
+def library_fragmentation_method_list(library):
+    item = library.copy()
+    item.update(
+        {
+            'fragmentation_method': ['chemical (DpnII restriction)', 'chemical (HindIII restriction)'],
+        }
+    )
+    return item
 
 def test_library_starting_quantity_post(testapp, library_starting_quantity):
     testapp.post_json('/library', library_starting_quantity)
+
+
+def test_library_fragmentation_method_string(testapp, library_fragmentation_method_string):
+    res = testapp.post_json('/library', library_fragmentation_method_string, expect_errors=True)
```

Comment:
You can assert status in one line using `testapp.post_json('/library', library_fragmentation_method_string, status=422)`

Reply:
done